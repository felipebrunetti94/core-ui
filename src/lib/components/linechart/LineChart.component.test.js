import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/linechart/__snapshots__"
});
