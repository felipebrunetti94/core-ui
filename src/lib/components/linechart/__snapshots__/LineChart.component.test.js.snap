// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots LineChart Default 1`] = `
<div>
  <h3>
    Vege-Lite line chart demo
  </h3>
  <div
    className="sc-vegachart sc-bdVaJa bDWFJH"
    id="vis"
  />
</div>
`;
